lib/antlr4-native/version.rb

```diff
@@ -1,3 +1,3 @@
 module Antlr4Native
-  VERSION = '2.0.1'
+  VERSION = '2.1.1'
```

Comment:
The patch version should be reset to zero when bumping the minor version.

```suggestion
  VERSION = '2.1.0'
```